tools/dataset_converters/dota_split/README.md

```diff
@@ -0,0 +1,136 @@
+# Preparing DOTA Dataset
+
+## download dota dataset
+
+The dota dataset can be downloaded from [DOTA](https://captain-whu.github.io/DOTA/dataset.html) or [OpenDataLab](https://opendatalab.org.cn/DOTA_V1.0).
+
+Please unzip the file and place it in the following structure.
+
+```none
+${DATA_ROOT}
+├── train
+│   ├── images
+│   │   ├── P0000.png
+│   │   ├── ...
+│   ├── labelTxt-v1.0
+│   │   ├── labelTxt
+│   │   │   ├── P0000.txt
+│   │   │   ├── ...
+│   │   ├── trainset_reclabelTxt
+│   │   │   ├── P0000.txt
+│   │   │   ├── ...
+├── val
+│   ├── images
+│   │   ├── P0003.png
+│   │   ├── ...
+│   ├── labelTxt-v1.0
+│   │   ├── labelTxt
+│   │   │   ├── P0003.txt
+│   │   │   ├── ...
+│   │   ├── valset_reclabelTxt
+│   │   │   ├── P0003.txt
+│   │   │   ├── ...
+├── test
+│   ├── images
+│   │   ├── P0006.png
+│   │   ├── ...
+
+```
+
+The folder ending with reclabelTxt stores the labels for the horizontal boxes and is not used when slicing.
+
+## Split dota dataset
+
+Script `tools/dataset_converters/dota_split/dota_split.py` can split and prepare DOTA dataset.
+
+shapely is required, please install shapely first by `pip install shapely`.
+
+```shell
+python tools/dataset_converters/dota_split/dota_split.py \
+    [--splt-config ${SPLIT_CONFIG}] \
+    [--data-root ${DATA_ROOT}] \
+    [--out-dir ${OUT_DIR}] \
+    [--ann-subdir ${ANN_SUBDIR}] \
+    [--phase ${DATASET_PHASE}] \
+    [--nproc ${NPROC}] \
+    [--save-ext ${SAVE_EXT}] \
+    [--overwrite]
+```
+
+**Description of all parameters**：
+
+- `--splt-config` : The split config for image slicing.
+- `--data-root`: Root dir of DOTA dataset.
+- `--out-dir`: Output dir for split result.
+- `--ann-subdir`: The subdir name for annotation. Defaults to `labelTxt-v1.0`.
+- `--phase`: Phase of the data set to be prepared. Defaults to `trainval test`
+- `--nproc`: Number of processes. Defaults to 8.
+- `--save-ext`: Extension of the saved image. Defaults to `png`
+- `--overwrite`: If the target folder already exists, whether to overwrite.
+
+Based on the configuration in the DOTA paper, we provide two commonly used split config.
+
+`./split_config/single_scale.json` means single scale split.
+`./split_config/multi_scale.json` means multi scale split.
+
```

Comment:
DOTA dataset usually uses the trainval set for training and the test set for [online evaluation](https://captain-whu.github.io/DOTA/evaluation.html), since most papers provide the results of online evaluation. If you want evaluate the model performance locally firstly, please split the train set and test set.